code/modules/power/gravitygenerator_vr.dm

```diff
@@ -395,7 +395,7 @@ GLOBAL_LIST_EMPTY(gravity_generators)
 		M.update_gravity(M.mob_has_gravity())
 		if(M.client)
```

Comment:
Hey could you remove this conditional? Both these procs already check for a client on their first line.

Reply:
sure why not